StylableSwiftUI/Classes/UIKit/Stylist+UIKit.swift

```diff
@@ -7,11 +7,18 @@
 
 import UIKit
 
+public enum TextCase {
+    case none
```

Comment:
Why do we have a `none` value?

Reply:
Um, seems silly I guess we could just have an optional value instead but IIRC it was to match the values output by sketch: https://github.com/design-ops/sketch-to-swiftUI/blob/master/lib/processTextStyles.js#L164